pkcs7.go

```diff
@@ -77,7 +75,11 @@ var (
 	OIDEncryptionAlgorithmECDSAP384 = asn1.ObjectIdentifier{1, 3, 132, 0, 34}
 	OIDEncryptionAlgorithmECDSAP521 = asn1.ObjectIdentifier{1, 3, 132, 0, 35}
 
-	// Encryption Algorithms
+	// Asymmetric Encryption Algorithms
```

Comment:
Both are defined here https://datatracker.ietf.org/doc/html/rfc8017 (and its obsoleted version https://datatracker.ietf.org/doc/html/rfc3447)

Reply:
I've added some references now. Not to all of them; will revisit sometime.